test/intl402/DisplayNames/prototype/of/length.js

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2025 André Bargull. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-Intl.DisplayNames.prototype.of
+description: >
+  Intl.DisplayNames.prototype.of.length is 1.
+info: |
+  ECMAScript Standard Built-in Objects
+
+  Every built-in function object, including constructors, has a "length" property
+  whose value is a non-negative integral Number. Unless otherwise specified,
+  this value is the number of required parameters shown in the subclause heading
+  for the function description. Optional parameters and rest parameters are not
+  included in the parameter count.
+
+  Unless otherwise specified, the "length" property of a built-in function object
+  has the attributes { [[Writable]]: false, [[Enumerable]]: false,
+  [[Configurable]]: true }.
+includes: [propertyHelper.js]
+features: [Intl.DisplayNames]
+---*/
+
+verifyProperty(Intl.DisplayNames.prototype.of, "length", {
```

Comment:
Can you instead update the other tests to use verifyPrimordialCallableProperty?

Reply:
`verifyPrimordialCallableProperty` doesn't support accessor methods, therefore I've instead used plain `verifyProperty` throughout this PR.